sdk/ml/azure-ai-ml/tests/component/unittests/test_command_component_entity.py

```diff
@@ -87,6 +101,34 @@ def test_command_component_entity(self):
 
         assert component_dict == yaml_component_dict
 
+    def test_command_component_with_additional_includes(self):
+        tests_root_dir = Path(__file__).parent.parent.parent
+        samples_dir = tests_root_dir / "test_configs/components/"
+
+        component = CommandComponent(
+            name="additional_files",
+            description="A sample to demonstrate component with additional files",
+            version="0.0.1",
+            command="echo Hello World",
+            environment=Environment(image="zzn2/azureml_sdk"),
+            # as sdk will take working directory as root folder, so we need to specify the absolution path
+            additional_includes=[
+                samples_dir / "additional_includes/assets/LICENSE",
+                samples_dir / "additional_includes/library.zip",
+                samples_dir / "additional_includes/library1",
+            ],
+        )
+        assert component._validate().passed, repr(component._validate())
+
+        with component._resolve_local_code() as code:
+            code_path: Path = code.path
+            assert code_path.is_dir()
+            assert (code_path / "LICENSE").is_file()
+            assert (code_path / "library.zip").is_file()
```

Comment:
I remember in 1.5 we will unzip this to a folder, is this expected to leave it as it is in v2?

Reply:
yes, it's expected. 
library is a folder and when customer write library.zip in additional_includes, we will zip library folder and upload it.